catalyst_voices/apps/voices/lib/pages/overall_spaces/spaces_overview_list_view.dart

```diff
@@ -27,6 +29,8 @@ class _SpacesListViewState extends State<SpacesListView> {
 
   @override
   Widget build(BuildContext context) {
+    final sessionBloc = context.watch<SessionCubit>();
+    final activeAccount = sessionBloc.account;
```

Comment:
context.watch will rebuild the whole widget every time the bloc state changes. We should prefer to only rebuild the widgets that depend on a given state. Also, we should try to extract from the state what we need and only rebuild widgets if the part of the state they are interested in changes.

Consider to wrap the
```
Offstage(
   offstage: activeAccount?.isAdmin == false,
```

with a `BlocSelector` instead of using context.watch